sample-apps/react-native/dogfood/src/hooks/useOrientation.ts

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from 'react';
+import { Dimensions } from 'react-native';
+
+export type Orientation = 'portrait' | 'landscape';
+
+const isPortrait = () => {
+  const dimensions = Dimensions.get('screen');
+  return dimensions.height >= dimensions.width;
+};
```

Comment:
small nitpick here... 

we can make this return `'portrait' : 'landscape'` ... to avoid the repetition inside the hook